scripts/test.sh

```diff
@@ -2,16 +2,29 @@
 
 echo "Starting Flame Tester"
 echo "---------------------"
+root_dir=$(pwd)
 for file in $(find . -type d -name "test"); do
   dir=$(dirname $file)
   cd $dir
   echo "Testing $dir"
-  flutter test
+  flutter test --coverage
   test_result=$?
   if [ $test_result -ne 0 ]; then
     exit $test_result
   fi
+
+  if [ -f ".min_coverage" ]; then
```

Comment:
Note to reviewers: Run coverage check only if the project has a minimum value set already, decided to do this way because we may have projects on the mono repo which don't make sense to check coverage